README.md

```diff
@@ -20,3 +20,6 @@ python hierarchical_agglomeration.py -f ../linux/include/linux/string.h -c outfi
 
 Replace with the appropriate header file location.
 Optionally add -d to find proximity betwen different tokens for further analysis.
+
+Below is an example graph of linux/types.h:
+![Alt](/images/Figure_1.png)
```

Comment:
Consider putting something more descriptive than `Alt` for the alternate text.  Perhaps something along the lines of `example of hiera... agglo.. on linux/types.h`.